test/tests/providers/global/loki.rules.test.yml

```
@@ -98,3 +98,29 @@ tests:
             exp_annotations:
               description: "Loki pod loki-compactor-676b8c897b-rq298 (namespace loki) sees 1 unhealthy ring members"
               opsrecipe: "loki/"
+  - interval: 1m
+    input_series:
+      - series: 'kube_pod_container_status_restarts_total{cluster_type="management_cluster", namespace="loki"}'
+        values: "0+0x20 0+100x20 0+0x120"  # 0 restarts after 20 minutes then we ramp up to 100 restarts in 20 minutes and then back to 0
```

This is not "100 restarts in 20 minutes" but "100 restarts per minute during 20 minutes".
Quite an unrealistic scenario IMO.
Also, the counter won't go back to 0 in normal situations (although that may also be a test we want to do).

So I'd rather go for:
```suggestion
        values: "0+0x20 0+5x20 100+0x120"  # 0 restarts after 20 minutes then we ramp up to 100 restarts in 20 minutes and then back to 0
```

Or maybe even: 
```
        values: "0+0x20 0+5x20 100+0x120 0+0x120"  # 0 restarts after 20 minutes then we ramp up to 100 restarts in 20 minutes, then stabilize, then test a counter reset to 0
```
(but for this one you may need to add an extra `eval_time` test point)

---

THanks Herve